frontend/src/app/pages/item-edit/item-edit.component.ts

```diff
@@ -82,9 +82,9 @@ export class ItemEditComponent implements OnInit {
     });
   }
 
-  getCollection() {
-    return this.categoryService.getAllCategories().subscribe((res) => {
-      this.collection = res.content;
+  getCategories() {
+    return this.categoryService.getAllCategories(0, 0, '', '').subscribe((res) => {
```

Comment:
Don't forget to add unsubscribe to every subscribe() function. It's very annoying, but should be done.
```
this.categoryService.getAllCategories(0, 0, '', '')
    .pipe(takeUntil(this.unsubscribe))
    .subscribe((res) => {...})
```